libs/libcups/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+# Copyright (C) 2017 Luiz Angelo Daros de Luca <luizluca@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cups
+PKG_VERSION:=1.5.4
```

Comment:
I didn't want to spend more time with (lib)cups that was only used to build hplip. I took cups as it was in oldpackages. But you are right, it's better to have an updated version that should be less prone to errors during the build. I bumped it to 2.2.3 and refreshed/removed some patches.